src/test/scala/pt/tecnico/dsi/neutron/QuotasSpec.scala

```diff
@@ -1,13 +1,63 @@
 package pt.tecnico.dsi.neutron
 
 import cats.effect.IO
+import pt.tecnico.dsi.neutron.models.Quota
+import pt.tecnico.dsi.openstack.keystone.models.Project
 
 class QuotasSpec extends Utils {
+  val withStubProject: IO[(NeutronClient[IO], String)] =
+    for {
+      keystone <- keystoneClient
+      dummyProject <- keystone.projects.create(Project("dummy", "dummy project", "default"))
+      neutron <- client
+    } yield (neutron, dummyProject.id)
+
   "Quotas service" should {
-    "get  quotas" in {
-      IO.pure {
-        assert(true)
+    "list quotas" in {
+      for {
+        (neutron, dummyProjectId) <- withStubProject
+        // Ensure there is at least one project with non-default quotas
+        _ <- neutron.quotas.update(dummyProjectId, Quota.Update(network = Some(30)))
+        quotas <- neutron.quotas.list.map(_._1).compile.toList
+        // Reset the quotas back to default values to ensure we can run this suite multiple times
+        _ <- neutron.quotas.delete(dummyProjectId)
+      } yield quotas should contain(dummyProjectId)
+    }
+    "get default quotas for a project" in withStubProject.flatMap { case (neutron, dummyProjectId) =>
+      neutron.quotas.getDefaults(dummyProjectId).idempotently { quota =>
+        quota.floatingip shouldBe 50
+        quota.network shouldBe 100
+        quota.port shouldBe 500
+        quota.rbacPolicy shouldBe 10
+        quota.router shouldBe 10
+        quota.securityGroupRule shouldBe 100
+        quota.securityGroup shouldBe 10
+        quota.subnet shouldBe 100
+        quota.subnetpool shouldBe -1
```

Comment:
Where are these values coming from?

Reply:
Might be more self-explanatory if we create an instance of a default quota to do comparisons with, and comment it with an explanation of the values (or a link to the docs).

Reply:
That is the correct approach. However these values are specific to our installation of Openstack, so that instance of Quota with default values would have to only exist in tests. I'll refactor it.